src/icons/index.ts

```diff
@@ -12,6 +12,10 @@ export { ReactComponent as ChevronRight } from './images/ChevronRightIcon.svg';
 export { ReactComponent as ChevronUp } from './images/ChevronUpIcon.svg';
 export { ReactComponent as Close } from './images/CloseIcon.svg';
 export { ReactComponent as Cog } from './images/CogIcon.svg';
+export { ReactComponent as Color0 } from './images/color-0.svg';
+export { ReactComponent as Color1 } from './images/color-1.svg';
+export { ReactComponent as Color2 } from './images/color-2.svg';
+export { ReactComponent as Color3 } from './images/color-3.svg';
```

Comment:
You shouldn't need any of these icons.  In the docs, you can import `Cog` and wrap in a `div` with `color` and `backgroundColor` set to your needs in each of the 4 use cases.

Reply:
I have changed the code to reuse the CogIcon with a custom bgColor :)